packages/ui-predicate-react/.storybook/preview-head.html

```diff
@@ -0,0 +1,81 @@
+<!-- Storybook will inject these tags to the iframe where your components are rendered.
+So, these won’t be loaded into the main Storybook UI. -->
+<style>
+  /* for demo purpose */
+  .ui-predicate.compound {
+    margin-left: 10px;
+  }
+</style>
+
+<style>
+  .custom-css .ui-predicate__main,
```

Comment:
should we move this CSS outside of both react & vuejs package?

Reply:
You decide.
We could leave it as is for now. And then in the next PR see if we can move it to a central place. I'm still working on getting the CSS to nicely build in.

Reply:
Actually... thinking about this. This is really bound to the storybook, right?
With every view implementation has its own repo, it also has its own storybook. So sharing the CSS between them kind of couples the individual packages in a way. On the other hand... this is a bit of repeating yourself (no so DRY - can I say WET? )

So what do you think?